webapp-django/crashstats/crashstats/templates/crashstats/crashes_per_user.html

```diff
@@ -0,0 +1,275 @@
+{% extends "crashstats_base.html" %}
+{% block site_css %}
+  {{ super() }}
```

Comment:
indentation is weird in this file, it's mostly 4-spaces but sometimes it's 2-spaces.